README.md

```diff
@@ -5,19 +5,25 @@
 A Gnome extension to show Nvidia GPU information
 (https://extensions.gnome.org/extension/1320/nvidia-gpu-stats-tool/)
 
-# Requirements
-nvidia-settings and nvidia-smi
+## Requirements
+The extension requires `nvidia-smi`, which should be included with the drivers.
+Additionally, `nvidia-settings` is supported as a data source.
 
-# Installation from git
+## Installation from git
     git clone https://github.com/ethanwharris/gnome-nvidia-extension.git
     cd gnome-nvidia-extension
     make
     make install
 
-Use `alt + f2`, `r`, `Enter` to restart the gnome shell (if this is not allowed then log out and log back in)
-Enable the extension in the gnome tweak tool.
+Restart gnome shell by logging out and back in.
```

Comment:
Could be rephrased to specify wayland and avoid any ambiguity:
```
- restart gnome shell to load the installed extension:
    - On Wayland, log out and log back in
    - On X11, (etc)
- Afterwards (etc)
```

Reply:
done!